[metricbeat/kubernetes] Remove fallback to the Node's limit for the pod 'usage.limit.pct' metrics (#40029)
elastic/beats: metricbeat/docs/fields.asciidoc

*`kubernetes.pod.cpu.usage.limit.pct`*::
+
--

> CPU usage as a percentage of the defined cpu limits sum of the pod containers. If one or more containers of the pod is unlimited, this metric will be missing entirely.

Reviewer: ```suggestion
CPU usage as a percentage of the defined cpu limits sum of the pod containers. If any container is missing a limit the metric is not emitted..
```

*`kubernetes.pod.memory.usage.limit.pct`*::
+
--

> Memory usage as a percentage of the defined memory limits sum of the pod containers. If one or more containers of the pod is unlimited, this metric will be missing entirely.

Reviewer: ```suggestion
Memory usage as a percentage of the defined memory limits sum of the pod containers. If any container is missing a limit the metric is not emitted.
```

*`kubernetes.pod.memory.working_set.limit.pct`*::
+
--

> Working set memory usage as a percentage of the defined limits sum of the pod containers. If one or more containers of the pod is unlimited, this metric will be missing entirely.

Reviewer: ```suggestion
Working set memory usage as a percentage of the defined limits sum of the pod containers. If any container is missing a limit the metric is not emitted.
```
